src/searchplugin.css

```diff
@@ -22,10 +22,32 @@
     margin-top: 4px;
 }
 
+.nk-search .form-control{
+    width: 250px;
+}
+.nk-search .error{
+    border: 1px solid #ccc;
+    animation: error-blink 2s;
+}
+@keyframes error-blink {
+  /* You could think of as "step 1" */
```

Comment:
what does this comment mean?

Reply:
That the animation goes from step 1: red, to step 2: grey again. I can explain it better I guess :)